src/core/connection.c

```diff
@@ -1965,6 +1966,8 @@ QuicConnStart(
     Connection->RemoteServerName = ServerName;
     ServerName = NULL;
 
+    Connection->State.IgnoreFixedBit = Configuration->Settings.GreaseQuicBitEnabled;
```

Comment:
Do we actually need the `IgnoreFixedBit` flag at all, or can we replace al usage with `Settings.GreaseQuicBitEnabled`?

Reply:
No, we don't need it. I'm gonna delete it in the next commit.